spark/client/channel/channel_test.go

```diff
@@ -58,6 +58,7 @@ func TestBasicChannelParsing(t *testing.T) {
 	assert.ErrorIs(t, err, sparkerrors.InvalidInputError)
 
 	cb, err = channel.NewBuilder(goodChannelURL)
+	assert.Nilf(t, err, "Should not have an error for a proper URL")
```

Comment:
`assert.NoError` is the right one to use here

Reply:
Done